manifests/JFrog/Conan/1.25.2.0.yaml

```diff
@@ -0,0 +1,16 @@
+Id: JFrog.Conan
+Publisher: JFrog
+Name: Conan
+Version: 1.25.2.0
+License: MIT
+InstallerType: inno
+LicenseUrl: https://github.com/conan-io/conan/blob/develop/LICENSE.md
+AppMoniker: 
```

Comment:
```suggestion
```

Reply:
I got into this kind of situation once. It seems that this line should be removed if it's void.